src/crouter/route.cr

```diff
@@ -58,7 +60,8 @@ module Crouter
       end
 
       if request.headers["Content-Type"]? == "application/x-www-form-urlencoded"
-        HTTP::Params.parse(request.body || "") do |name, value|
+        body = request.body.try &.gets_to_end
```

Comment:
could you use parens here? I'm trying to maintain the style that every method call with params has parens except for "special" ones like puts